scripts/travis.sh

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env bash
```

Comment:
@siyangy  I fixed your concern, and remove the travis.sh file. Now only the travis ci configure file is needed.